radix-engine-constants/src/lib.rs

```diff
@@ -55,5 +55,11 @@ pub const DEFAULT_MAX_SUBSTATE_READS_PER_TRANSACTION: usize = 20_000;
 /// The default maximum substates writes count per transaction.
 pub const DEFAULT_MAX_SUBSTATE_WRITES_PER_TRANSACTION: usize = 5_000;
 
+/// The default maximum substates read size.
+pub const DEFAULT_MAX_SUBSTATE_READ_SIZE: usize = 500 * 1024;
+
+/// The default maximum substates write size.
+pub const DEFAULT_MAX_SUBSTATE_WRITE_SIZE: usize = 10 * 1024;
```

Comment:
Can we add a `TODO: Apply this limit in create_node too`?

Reply:
done